python/cugraph/link_prediction/jaccard.py

```diff
@@ -16,7 +16,7 @@
 import cudf
 
 
-def jaccard(input_graph, first=None, second=None):
+def jaccard(input_graph, vertex_pair=None):
```

Comment:
Please add a note for this API change in the description of the PR

Reply:
added